src/shop/_components/rewardsSection/RewardsSection.tsx

```diff
@@ -0,0 +1,57 @@
+import React, { useCallback } from 'react'
+import { useTranslation } from 'react-i18next';
+import { FlatList } from 'react-native';
+
+import { Icon, Reward, Typography } from '../../../_components';
+import { REWARD_TILE_WIDTH } from '../../../_components/reward/style';
+import { TReward } from '../../_models/reward';
+import { TFilterRewardSections, useGetRewards } from '../../_queries/useGetRewards';
+import * as Styled from './style';
+
+type TProps = {
+  filter: TFilterRewardSections;
+  horizontal?: boolean;
+  title: string;
+}
+export const RewardsSection = ({ horizontal, filter, title }: TProps) => {
+  const { data: rewards, isLoading } = useGetRewards({ itemsPerPage: horizontal ? 20 : 3, section: filter });
+  const { t } = useTranslation();
+
+  const onPressReward = useCallback((_reward: TReward) => {
+  }, []);
+
+  // We need to have 2 or more results to display the section
+  if (!isLoading && !(rewards?.pages[0]?.member.length >= 2)) return null;
+
+  return (
+    <Styled.Container>
+      <Styled.Header>
+        <Typography fontStyle='bold' size='large'>{title}</Typography>
+        <Styled.ShowMoreButton activeOpacity={0.8} onPress={() => { }}>
+          <Typography color="primary.800" fontStyle='bold' size="small">{t('SHOP.SHOW_MORE')} </Typography><Icon color="primary.800" name="ArrowRight" />
+        </Styled.ShowMoreButton>
+      </Styled.Header>
+
+      {horizontal ? <>
```

Comment:
hier gebruik je dan horizontal ipv tile of list

Reply:
Omdat beide sections nu ook eenmaal een list zijn 😄  Een horizontale list met tiles, of een verticale met listItems... Maar in se is het enige verschil hier horizontaal of verticaal, vandaar die prop